ci/script.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set -ex
+
+case "$GIMLI_JOB" in
+    "test")
+        cargo clean
+        cargo build $GIMLI_PROFILE
+        cargo test $GIMLI_PROFILE
+        ;;
+
+    "examples")
+        cargo build $GIMLI_PROFILE
+        case "$TRAVIS_OS_NAME" in
+            "osx")
+                with_debug_info=$(find ./target/debug -type f | grep DWARF | grep gimli | head -n 1)
+                ;;
+            "linux")
+                with_debug_info=$(find ./target/debug -type f -perm -100 | grep gimli | head -n 1)
+                ;;
+            *)
+                echo "Error! Unknown \$TRAVIS_OS_NAME: $TRAVIS_OS_NAME"
+                exit 1
+        esac
+        cargo run --example dwarfdump -- "$with_debug_info" > /dev/null
+        ;;
+
+    "doc")
+        cargo doc
+        ;;
+
+    "alloc")
+        test "$TRAVIS_RUST_VERSION" == "nightly"
+        cargo clean
+        cargo build --no-default-features --features alloc $GIMLI_PROFILE
+        ;;
+
+    "bench")
+        cargo bench
+        ;;
+
+    "coverage")
+        pip2 install -v 'travis-cargo<0.2' --user
+        TRAVIS_CARGO_NIGHTLY_FEATURE= travis-cargo coveralls --no-sudo --verify
```

Comment:
Looks like this line is causing the build error. We should ignore any error on this line because we're only using it to install kcov. This wasn't a problem before because we were doing coverage in the after_success commands.

Or we could switch to tarpaulin.